src/extension.ts

```diff
@@ -53,9 +44,9 @@ export async function activate(context: vscode.ExtensionContext) {
 
   const clangd: vscodelc.Executable = {
     command: clangdPath,
```

Comment:
by reading the LSP client [code](https://github.com/microsoft/vscode-languageserver-node/blob/635ab1fe6f8c57ce9402e573d007f24d6d290fd3/client/src/main.ts#L468), it seems that it will try to find a working directory (default is the workspacefolder) as the command `cwd`, that might explain why the relative path was working prior to the auto-install patch.

Reply:
Yeah, I understand why it happened to work before, but I don't think we should continue to support it - it would be a bit of work (since we access it in a few places now) and seems pretty fragile.

Reply:
Now I understood what you meant by "(default is the workspacefolder)", maybe this is actually worth preserving. In a followup patch...